db/migrate/20230607124047_add_oauth_attributes_to_staff.rb

```diff
@@ -0,0 +1,8 @@
+class AddOauthAttributesToStaff < ActiveRecord::Migration[7.0]
+  def change
+    change_table :staff, bulk: true do |t|
+      t.string :provider
```

Comment:
I'm wondering if there's any benefit to storing these? We don't actually use them for anything, especially `provider` which we know there will only ever be one.

Reply:
Hmmm; I'm not sure if we could need these going forward. Will we ever want to look up a provider for a staff member? Keeping it on there seems low cost.

Reply:
There will only ever be one provider, `azure_ad` so we can't definitely get rid of `provider` I reckon. Keeping `uid` seems more reasonable, but I'd be tempted to rename it to `azure_ad_uid` or something because `uid` on its own is a bit unclear what it is.